src/Config/BaseConfig.php

```diff
@@ -17,12 +17,13 @@ public function __construct(SystemConfigService $systemConfig)
     }
 
     /**
-     * @param string $param
+     * @param string      $param
+     * @param string|null $salesChannelId
      *
      * @return mixed
      */
-    protected function config(string $param)
+    protected function config(string $param, ?string $salesChannelId = null)
```

Comment:
should we use question mark if we declare argument as nullable, using equal mark?

Reply:
If you skip the default value then you would need to pass this argument to the function, otherwise you will get
`Uncaught ArgumentCountError: Too few arguments to function `

calling function like `foo(null)` is awkward in my opinion

Reply:
![image](https://user-images.githubusercontent.com/4511883/124756236-881ba400-df2c-11eb-885f-8447af34b0e2.png)